packages/bpk-component-infinite-scroll/src/onItemsFetch.js

```diff
@@ -18,6 +18,11 @@
 
 /* @flow */
 
+export type OnItemsFetchFunction = (
```

Comment:
I think we should remove this function and replace it whit an `ArrayDataSource`, something like this. 

```
class ArrayDataSource extends DataSource {
   constructor(elementsArray) {
      this.elementsArray = elementsArray;
   }
   
   fetchItems(index, nElements) {
     if (n <= 0) {
       return resolve([]);	     
     }
     return resolve(elementsArray.slice(0, index + nElements));
  }
}
```

Can be defined in the same file you're defining the `DataSource` class